controllers/platformnetwork_controller.go

```diff
@@ -1325,3 +1332,38 @@ func (r *PlatformNetworkReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&starlingxv1.PlatformNetwork{}).
 		Complete(r)
 }
+
+// Verify whether we have annotation restore-in-progress
+func (r *PlatformNetworkReconciler) checkRestoreInProgress(instance *starlingxv1.PlatformNetwork) bool {
+	restoreInProgress, ok := instance.Annotations[cloudManager.RestoreInProgress]
+	if ok && restoreInProgress != "" {
+		return true
+	}
+	return false
+}
+
+// Update status and clear annoatation
+func (r *PlatformNetworkReconciler) RestorePlatformNetworkStatus(instance *starlingxv1.PlatformNetwork) error {
+	r.ReconcilerEventLogger.NormalEvent(instance, common.ResourceUpdated, "Status without reconcilation: %t", instance.Status.InSync)
+	annotation := instance.GetObjectMeta().GetAnnotations()
+	if annotation != nil {
+		config, ok := annotation["kubectl.kubernetes.io/last-applied-configuration"]
```

Comment:
This should use `cloudManager.RestoreInProgress` annotation key.

Reply:
Changes are done